TramsDataApi/Factories/TrustResponseFactory.cs

```diff
@@ -1,14 +1,48 @@
 using System.Collections.Generic;
+using System.Linq;
 using TramsDataApi.DatabaseModels;
 using TramsDataApi.ResponseModels;
 
 namespace TramsDataApi.Factories
 {
     public static class TrustResponseFactory
     {
-        public static TrustResponse Create(Group group)
+        public static TrustResponse Create(Group group, Trust ifdTrustData, List<Establishment> establishments)
         {
-            var ifdDataResponse = new IFDDataResponse();
+            IFDDataResponse ifdDataResponse;
+            if (ifdTrustData == null)
+            {
+                ifdDataResponse = null;
+            }
+            else
+            {
+                ifdDataResponse = new IFDDataResponse
+                {
+                    TrustOpenDate = ifdTrustData.TrustsTrustOpenDate.ToString(),
+                    LeadRSCRegion = ifdTrustData.LeadRscRegion,
+                    TrustContactPhoneNumber = ifdTrustData.TrustContactDetailsTrustContactPhoneNumber,
+                    PerformanceAndRiskDateOfMeeting = ifdTrustData.TrustPerformanceAndRiskDateOfMeeting.ToString(),
+                    PrioritisedAreaOfReview = ifdTrustData.TrustPerformanceAndRiskPrioritisedForAReview,
+                    CurrentSingleListGrouping = ifdTrustData.TrustPerformanceAndRiskSingleListGrouping,
+                    DateOfGroupingDecision = ifdTrustData.TrustPerformanceAndRiskDateOfGroupingDecision.ToString(),
+                    DateEnteredOntoSingleList = ifdTrustData.TrustPerformanceAndRiskDateEnteredOntoSingleList.ToString(),
+                    TrustReviewWriteup = ifdTrustData.TrustPerformanceAndRiskTrustReviewWriteUp,
+                    DateOfTrustReviewMeeting = ifdTrustData.TrustPerformanceAndRiskDateOfMeeting.ToString(),
```

Comment:
Do we know what format the dates are in? `DateTime.ToString` has an optional format as a parameter, so if we know they should be in a specific format (I assume `dd/mm/yyyy`), then we should specify it here